File: app/src/main/java/com/example/android/moviesremake/retrofit/model/MovieRetrofitReview.java
package com.example.android.moviesremake.retrofit.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by PepovPC on 1/16/2018.
 */

public class MovieRetrofitReview {

    @SerializedName("author")
    private String autorName;

    @SerializedName("content")
    private String autorComment;


    public MovieRetrofitReview(String autorName, String autorComment){
        this.autorName = autorName;
        this.autorComment = autorComment;
    }


    public String getAutorName() {
        return autorName;
    }

    public void setAutorName(String autorName) {
        this.autorName = autorName;
    }

    public String getAutorComment() {
        return autorComment;
    }

    public void setAutorComment(String autorName) {
        this.autorComment = autorComment;
    }


}

File: app/src/main/java/com/example/android/moviesremake/retrofit/model/MovieRetrofitReview.kt
package com.example.android.moviesremake.retrofit.model

import com.google.gson.annotations.SerializedName

/**
* Created by PepovPC on 1/16/2018.
*/

class MovieRetrofitReview(
        @field:SerializedName("author") var autorName: String?,
        @field:SerializedName("content") private var autorComment: String?) {

    fun getAutorComment(): String? {
        return autorComment
    }

    fun setAutorComment(autorName: String) {
        this.autorComment = autorComment
    }


}
